resources/components/ImageCard.vue

```diff
@@ -36,6 +36,14 @@
 						v-bind:confirmed="suggestion.confirmed"
 						v-on:click="toggleTagConfirmation( suggestion )"
 					/>
+
+					<!-- Add custom tag button -->
+					<suggestion
```

Comment:
@AnneTee what do you think about handling the "add custom tag" button as a variant of `<Suggestion>`? I also considered using a raw button but this seemed like the best way to ensure visual parity between this and the other suggested labels. 

The potential downside of re-purposing the `<Suggestion>` component in this way is that we'd need to come up with an alternative way to handle the icon (here we want a + icon that's always visible, as opposed to a check icon that only shows up when the suggestion is confirmed).

Let me know if you have a preference and I'll try to finalize this PR (get the icon to display here, basically).

Reply:
I could see the argument for either method, but I think it makes more sense to use a Button and style it like a Suggestion. A Suggestion is basically just a button with a specific icon and two states, and the Add Tag button doesn't need that icon or the states. You should be able to reuse styles [here](https://github.com/wikimedia/mediawiki-extensions-MachineVision/commit/1266b6a78f0f254a3d29a03738247bd8fb5eafc9#diff-70c29f83eedc43cea0556e33274c157bR116) (minus all the OOUI overrides of course!) to achieve parity.

Reply:
Ok – done. Please take a look and confirm that things are styled correctly.